src/plugins/recordTypes/exhibition/fields.js

```diff
@@ -537,6 +537,30 @@ export default (configContext) => {
             },
           },
         },
+        publishToList: {
+          [config]: {
+            view: {
+              type: CompoundInput,
+            },
+          },
+          publishTo: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.exhibition_common.publishTo.name',
```

Comment:
This should be `exhibitions_common` for consistency with other message ids in the file.

Reply:
Thanks, just pushed a commit to resolve this. I'll be sure to keep my eye out for this in the future.